src/main/java/com/immunopass/entity/VoucherEntity.java

```diff
@@ -31,30 +28,56 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(name = "voucher_code")
```

Comment:
No need of @column annotation as the mapping is handled by spring.jackson.property-naming-strategy=SNAKE_CASE property.

Reply:
cool